tests/test_dbos.py

```diff
@@ -334,6 +337,13 @@ def test_transaction(var2: str) -> str:
         txn_counter += 1
         return var2 + str(rows[0][0])
 
+    @DBOS.transaction()
+    def test_transaction_return_none() -> None:
```

Comment:
Should we add a test_step_returns_none test? Or is the test Qian added for the Outcome None issue sufficient?

Reply:
Yeah, Qian's test already does that